blues/sparrow-reference-web-app
# fix: adding a maxlength for sensor name & location

# Problem Context

> The name and location are both editable from the details tab and both are optional with a maximum length. Name has a max length of 49 characters, The location is intended to be in OLC format and has a maximum length of 15 characters. 

## Changes

Simple implementation of this with two `maxlength` attributes.

## Testing

- Run the app.
- Try to type more than 49 characters in Name, or 15 characters in Location. You should not be able to.

## Ticket(s)

https://www.pivotaltracker.com/story/show/181159368
